pennylane/qinfo/transforms.py

```diff
@@ -663,6 +667,12 @@ def circ(params):
             [2.16840434e-18, 2.81967252e-01]]]))
 
     """
+    warnings.warn(
+        "pennylane.qinfo.classical_fisher is being migrated to a different module and will "
+        "removed in version 0.39. Instead, use pennylane.gradient.classical_fisher.",
+        qml.PennyLaneDeprecationWarning,
+    )
+
     new_qnode = _make_probs(qnode)
```

Comment:
Optional, non-blocking:

This is a really minor comment since this function will be removed soon anyway, but you could also just call `qml.gradients.classical/quantum_fisher` in here rather than keeping the full implementation (unless of course there are circular dependencies).